openslides/users/migrations/0004_groups.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.7 on 2016-08-01 14:54
+from __future__ import unicode_literals
+
+from django.contrib.auth.models import Group
```

Comment:
Is it possible to get Group also from app registry and not by import? See line 20.